test/AlgebraicGeometry/Schemes/AffineVariety.jl

```diff
@@ -0,0 +1,3 @@
+@testset "affine variety" begin
+  P, (x,y) = polynomial_ring(QQ, [:x,:y])
+end
```

Comment:
There seems to be something missing here -- maybe editor not saved before commit?